README.md

```diff
@@ -6,7 +6,7 @@ The UBC Launch Pad design team and relevant documentation, links, and resources.
 
 ### Meeting Notes
 
-Meeting notes can be found in the repository [wiki](https://github.com/ubclaunchpad/design/wiki).
+Design team meeting notes can be found in the repository [wiki](https://github.com/ubclaunchpad/design/wiki).
```

Comment:
This is no longer valid - let's get rid of this entire meeting notes section for now (it should be covered by the `Design Workflow` link)